docs-ref-conceptual/azure-cli-vm-tutorial-5.md

```diff
@@ -45,21 +57,54 @@ for command-line tools, it helps with human readability and can be used with cus
 In order to use command-line tools, change the command to remove the custom JSON keys and output as `tsv`. The `read` command processes this style of output by loading results into multiple variables. Since two values are displayed on separate lines, the `read` command
 delimiter must be set to the empty string rather than the default of non-newline whitespace.
 
-```azurecli
+# [Bash](#tab/bash)
+
+```azurecli-interactive
 read -d '' ipId subnetId <<< $(az network nic show \
   --ids $nicId \
-  --query '[ipConfigurations[].publicIpAddress.id, ipConfigurations[].subnet.id]' \
+  --query '[ipConfigurations[].publicIPAddress.id, ipConfigurations[].subnet.id]' \
   -o tsv)
 ```
 
+# [PowerShell](#tab/powershell)
+
+```azurecli-interactive
+$output = az network nic show --ids $nicId --query "[ipConfigurations[].publicIPAddress.id, ipConfigurations[].subnet.id]" -o tsv
+
+# Split the output into two variables
+$ipId, $subnetId = $output -split "`t"
+```
+
+---
+
+If you don't want to use the Bash `read` or PowerShell `-split` command, you can set each variable separately.
```

Comment:
@v-abdriaz , not everyone wants to mix Bash commands and Azure CLI, but I just stuck this in here as I used it for testing.  This isn't something I'd expect you to add.